src/Autosuggest.js

```diff
@@ -515,7 +516,8 @@ export default class Autosuggest extends Component {
       theme,
       getSuggestionValue,
       alwaysRenderSuggestions,
-      highlightFirstSuggestion
+      highlightFirstSuggestion,
+      containerProps
```

Comment:
```suggestion
      containerProps = null
```

Reply:
Agreed, although in fact, I'd suggest making them an empty object rather than just `null`.